dev/devicelab/lib/framework/devices.dart

```diff
@@ -193,6 +194,20 @@ abstract class Device {
   /// Stop a process.
   Future<void> stop(String packageName);
 
+  /// Wait for the device to become ready.
+  Future<void> awaitDevice();
+
+  Future<void> uninstallApp() async {
+    await flutter('install', options: <String>[
+      '--uninstall-only',
+      '-d',
+      deviceId]);
+
+    sleep(const Duration(seconds: 2));
```

Comment:
nit: sleep() will completely block the isolate, whereas using Future.delayed() will allow other asynchronous work to run.

```suggestion
    await Future.delayed(const Duration(seconds: 2));
```